assets/scripts/streets/StreetMetaData.jsx

```diff
@@ -47,6 +54,9 @@ class StreetMetaData extends React.Component {
         <StreetWidth street={this.state.street} readOnly={this.props.readOnly} />
         <span id='street-metadata-author'>{author}</span>
         <span id='street-metadata-date'>{formatDate(this.state.street.updatedAt)}</span>
+        <a id='street-metadata-map' onClick={this.onClick.bind(this)}><u>Geolocation!</u></a>
```

Comment:
You should do `.bind(this)` in the constructor. The `.bind()` function incurs a slight performance hit, so doing this in the constructor only happens once. Doing it in the `render()` means it has to run every render, which slows down the app.